### DISCUSSION TITLE
Message highlights taking priority over user highlights

### DISCUSSION THREAD

**Discussion by LosFarmosCTL** [Chatterino/chatterino2#3865] [Ideas] [answered]
Was there a specific reason to make user highlights take priority over message/phrase highlights?

IMO it would make more sense the other way around, highlight all messages from one user, but i.e. if that one user pings me I'd still want that to take priority.

**Comment by pajlada** [comment]
As mentioned in #3871 and #3872 , it's clear to me that we need to have an ability to customize the highlight order.

@TigerWarz posted this image in his discussion:
![image](https://user-images.githubusercontent.com/962989/180454298-c85a4852-5864-4547-81eb-7a2957b0ee71.png)
and I think that combined with a similar priority "slider" for each highlight would fulfill everyone's wishes.

On the backend, this would be easily solvable.

**Comment by pajlada** [comment, marked as answer]
Message highlight will take priority over user highlights after https://github.com/Chatterino/chatterino2/pull/4303
